src/app/services/search.service.ts

```diff
@@ -54,14 +54,20 @@ export class ResearchStudySearchEntry {
    */
   resource: fhirclient.FHIR.Resource;
   search?: Search;
+  provider: String;
   private cachedSites: fhirpath.FHIRResource[] | null = null;
   private containedResources: Map<string, fhirpath.FHIRResource> | null = null;
   dist: number | undefined;
-  constructor(public entry: BundleEntry, private distService: DistanceService, private zipCode: string) {
+  constructor(
+    public entry: BundleEntry,
+    private distService: DistanceService,
+    private zipCode: string,
+    provider: string
+  ) {
     this.resource = this.entry.resource;
     this.search = this.entry.search;
     console.log(this.entry);
-
+    this.provider = provider;
```

Comment:
You can mark `provider: string` `public` in the constructor and remove the `provider: string;` declaration from the class and the `this.provider = provider;` line from the constructor. Also, the string type should be `string` and not `String`.